test/extended/router/metrics.go

```diff
@@ -91,9 +91,7 @@ var _ = g.Describe("[sig-network][Feature:Router]", func() {
 				context.TODO(),
 				"prometheus-k8s",
 				&authenticationv1.TokenRequest{
-					Spec: authenticationv1.TokenRequestSpec{
-						Audiences: []string{"https://kubernetes.default.svc"},
-					},
+					Spec: authenticationv1.TokenRequestSpec{},
```

Comment:
/cc @tkashem 

I am not sure if this is the correct thing to do, I am only sure that hardcoding is not the correct thing to do. I tested it locally and it seemed to work.